docs/java-application-development/development-environment-setup.md

```diff
@@ -1,183 +1,161 @@
 # Development Environment Setup
 
-This document describes how to set up the development environment for Kura, which consists of the following components:
-
-* JVM (Java JDK SE 8)
-
-* Eclipse IDE
-
-* Kura Workspace setup
+In this document we'll cover the required steps to setup the Development Environment for contributing to the Eclipse Kura project. If, instead you want to develop applications or bundles running on Kura refer to the [Kura Workspace setup guide](./kura-workspace-setup.md).
```

Comment:
> If, instead you want

If, instead, you want

Reply:
Addressed in d2d26aa76ea690f92675a710ebf81a99c1857ada